src/python_bring_api/bring.py

```diff
@@ -293,3 +297,58 @@ def removeItem(self, listUuid: str, itemName: str) -> Response:
         except RequestException as e:
             _LOGGER.error(f'Exception: Could not remove item {itemName} from list {listUuid}:\n{traceback.format_exc()}')
             raise BringRequestException(f'Removing item {itemName} from list {listUuid} failed due to request exception.') from e
+
+
+    def notify(self, listUuid: str, notificationType: str, itemName: str = None) -> Response:

```

Comment:
Why is the message called item name?

Reply:
It is basically a text input field where you can write whatever you want but the description for the input field in the App is `Artikelname` and the recipient gets the message "Besorg bitte `Artikelname `, so i thought it should be clear what kind of message this field is for

Reply:
Thanks, I see!